hack/windows/Install-OVS.ps1

```diff
@@ -32,7 +32,7 @@ Param(
 )
 
 $ErrorActionPreference = "Stop"
-$OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-2.15.2-antrea.0-win64.zip"
+$OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-2.16.7-antrea.0-win64.zip"
```

Comment:
I am wondering how do we generate this OVS? anyway to check what we have in the download site?

Reply:
Yes the testbed has been upgraded. I will run all windows CI jobs to verify new ovs.

Reply:
any idea why it's v2.16.7 on Windows but not the same version as Linux? https://github.com/antrea-io/antrea/pull/4742